commandapi-platforms/commandapi-bukkit/commandapi-bukkit-nms/commandapi-bukkit-nms-common/src/main/java/dev/jorel/commandapi/nms/NMS_Common.java

```diff
@@ -92,6 +94,18 @@
  */
 public abstract class NMS_Common extends CommandAPIBukkit<CommandSourceStack> {
 
+	private static final Field commandDispatcher;
+
+	static {
+		Field temporary;
+		try {
+			temporary = MinecraftServer.class.getDeclaredField("vanillaCommandDispatcher");
+		} catch (Exception e) {
+			temporary = null;
+		}
+		commandDispatcher = temporary;
+	}
```

Comment:
A field that could be null that doesn't have `@Nullable` or wrapped in an `Optional`! 🫢